node/src/components/block_validator/state.rs

```diff
@@ -178,6 +177,30 @@ impl BlockValidationState {
         (state, None)
     }
 
+    fn validate_transaction_category_counts(
+        block: &ProposedBlock<ClContext>,
+        config: &TransactionConfig,
+    ) -> Result<(), ()> {
+        if block.standard_count() > config.block_max_standard_count as usize {
+            warn!("too many standard transactions");
+            return Err(());
+        }
+        if block.auction_count() > config.block_max_auction_count as usize {
+            warn!("too many staking transactions");
+            return Err(());
+        }
+        if block.install_upgrade_count() > config.block_max_install_upgrade_count as usize {
+            warn!("too many install_upgrade transactions");
+            return Err(());
+        }
+        if block.mint_count() > config.block_max_mint_count as usize {
+            warn!("too many transfers");
```

Comment:
Please update all places that relate to `mint/transfers` to be consistent with the renamed transaction categories.